docs/sources/operators-guide/configuring/reference-configuration-parameters/index.md

```diff
@@ -383,6 +403,10 @@ grpc_tls_config:
 # CLI flag: -server.log-source-ips-regex
 [log_source_ips_regex: <string> | default = ""]
 
+# Optionally log requests at info level instead of debug level.
+# CLI flag: -server.log-request-at-info-level-enabled
+[log_request_at_info_level_enabled: <boolean> | default = false]
```

Comment:
I'm not not a big fan of adding another basic option, just coming in because of a dependency and not a real Mimir use case. I would hide it.

Reply:
@pracucci thanks, will try.

Reply:
@pracucci how can I hide it?

Reply:
Let's discuss options on Slack.